client/src/pages/AddTrainee.js

```diff
@@ -50,6 +52,13 @@ export const AddTrainee = () => {
 					value={fullname}
 					onChange={(e) => setFullName(e.target.value)}
 				/>
+				<label htmlFor="cohortId">Cohort Id </label>
+				<input
+					type="number"
+					placeholder="cohort Id"
+					value={cohortId}
+					onChange={(e) => setCohortId(e.target.value)}
+				/>
```

Comment:
you could have use inputefiled component

Reply:
It's just because when I want to use that component, I should write the same amount of code

Reply:
Using the component will add the label for you and will eventually add consistent styling. Definitely use a component if you have one.
You need an id on the input for the label to be associated with it.